src/controllers/controllerdebug.h

```diff
@@ -10,7 +10,8 @@ class ControllerDebug {
   public:
     // Any debug statement starting with this prefix bypasses the --logLevel
     // command line flags.
-    static constexpr const char* kLogMessagePrefix = "CDBG";
+    static constexpr const char kLogMessagePrefix[] = "CDBG";
+    static constexpr int kLogMessagePrefixLength = sizeof(kLogMessagePrefix);
```

Comment:
Length is misleading, because length doesn't include the null terminator:

```
static constexpr int kLogMessagePrefixLength = sizeof(kLogMessagePrefix) - 1;
```

This requires to modify the code where kLogMessagePrefixLength is actually used. The `+ 1` there was not referring to the null terminator but to another character following the "CDBG" characters.